Sources/Sentry/include/SentryPerformanceTracker.h

```diff
@@ -0,0 +1,75 @@
+#import "SentrySpanStatus.h"
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Tracks performance synchronizing span with its childs.
+ * A span will be finished only when all its children are finished.
+ * This class works on thread context, spans created in different
+ * threads are not related.
+ */
+@interface SentryPerformanceTracker : NSObject
```

Comment:
Thanks for the code docs on this class. It helps a lot to understand what it is doing without having to read the implementation. 👏